docs/docs/how-to/functions/getting-started.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Getting Started
```

Comment:
I feel like this is missing any info about deploying in Cloud. I had a paragraph in G-docs about that.

Reply:
I put the paragraph on the overview page (in the mansion PR https://github.com/gatsby-inc/mansion/pull/11587)